base.mk

```diff
@@ -116,7 +116,7 @@ BUILD ?= release
 CFLAGS += -ffunction-sections -fdata-sections
 LDFLAGS += -Xlinker --gc-sections
 ifeq ($(BUILD), debug)
-CFLAGS += -O0 -g -DDEBUG
+CFLAGS += -Og -g -DDEBUG
```

Comment:
-1 

I see your point here, but I'm afraid this has hurt us quite a few times in the past already.
-Og enables optimizations that we don't want to have in place most of the times when debugging.

What are you missing when -O0 is place more specifically, please?